operator-metadata/container.yaml

```
@@ -6,7 +6,7 @@ platforms:
   - s390x
   - aarch64
 operator_manifests:
-  enable_digest_pinning: false
-  enable_repo_replacements: false
+  enable_digest_pinning: true
+  enable_repo_replacements: true
```

This will require the repo and packing mapping information as well, something like below. The `package_mappings` should be an array of Brew packages mapped to its release repository.

```suggestion
  enable_repo_replacements: true
  enable_registry_replacements: true
  repo_replacements:
    - registry: registry-proxy.engineering.redhat.com
      package_mappings:
        package-name-container: amq-streams/package-image-rhel9
```

---

@MikeEdgar do you have a reference for where these settings come from?

---

@k-wall , it's in the OSBS documentation for digest pinning and repo replacements [1]

[1] https://osbs.readthedocs.io/en/latest/users.html#replacing-pullspecs